pycbc/strain/strain.py

```diff
@@ -200,7 +200,42 @@ def from_cli(opt, dyn_range_fac=1, precision='single',
     gating_info = {}
 
     injector = InjectionSet.from_cli(opt)
+    # Read noiseless injections from frame file
+    if opt.injection_frame_cache or opt.injection_frame_files or \
```

Comment:
This section is nearly a complete duplication with the only change really being a couple argument names. Could this be combined or modularized?